src/java.base/share/classes/java/util/HexDigits.java

```diff
@@ -64,6 +102,23 @@ final class HexDigits implements Digits {
     private HexDigits() {
     }
 
+    /**
+     * Combine two hex shorts into one int based on big endian
+     */
+    static int digit(int b0, int b1) {
```

Comment:
The method name `digit` is not very informative.
As it is package private it should be more descriptive and describe the parameters and result.
Perhaps `packDigits` or `encodeAndPackHexDigits`.

Reply:
can you create a new issue and i continue improving?

Reply:
> can you create a new issue and i continue improving?

Filed: https://bugs.openjdk.org/browse/JDK-8311207

Reply:
@liach @RogerRiggs continue discussion in [PR 14745](https://github.com/openjdk/jdk/pull/14745)